core-management/core-management-subsystem/src/main/java/org/wildfly/extension/core/management/deployment/UnstableApiAnnotationAttachments.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright The WildFly Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.wildfly.extension.core.management.deployment;
+
+import org.jboss.as.server.deployment.AttachmentKey;
+import org.wildfly.unstable.api.annotation.classpath.runtime.bytecode.ClassInfoScanner;
+
+class UnstableApiAnnotationAttachments {
+    public static final AttachmentKey<ClassInfoScanner> UNSTABLE_API_ANNOTATION_SCANNER = AttachmentKey.create(ClassInfoScanner.class);
+
+    public static final AttachmentKey<Boolean> UNSTABLE_API_ANNOTATIONS_SCANNED = AttachmentKey.create(Boolean.class);
```

Comment:
Do these need to be public?  Generally, attachment keys are kept as private as possible.